docs/DeveloperGuide.adoc

```diff
@@ -374,7 +375,7 @@ _{More to be added}_
 .  Upon user input, PalPay should execute tasks (and display results) within 1 second.
 .  Users should have (and be able to specify) default options such that they can issue minimal commands for common tasks.
 .  Should be quick and efficient, with each user session lasting no longer than 3 minutes to effectively manage his current financial state.
-.  The application shouldn't make users feel defeated, but rather empowered in managing their finances.
+.  Should not make users feel defeated, but rather empowered in managing their finances.
```

Comment:
Overall NFRs are well written, except possibly for this one? How/When specifically will a user will defeated when using the application? Maybe it would be if the UI is too difficult to navigate? Thus maybe this NFR can be broken down into further sub-requirements. Good job!